cohorts/utils.py

```diff
@@ -185,3 +185,10 @@ def get_logger(name, level=logging.INFO):
         logger.handlers = []
     logger.setLevel(level)
     return logger
+
+def set_attributes(obj, additional_data):
+    if additional_data is not None:
+        for key, value in additional_data.items():
+            if hasattr(obj, key):
+                raise ValueError("Key %s in additional_data already exists in this object" % key)
```

Comment:
Seems like you might run into this a fair amount, at least at the patient level. E.g. if the additional_data includes a variable like "hla_alleles" or "deceased" which has already been set. Is this the intended behavior?